static/js/protein/chart.ts

```diff
@@ -241,8 +345,8 @@ export function drawTissueScoreChart(id: string, data: ProteinStrData[]): void {
 
   //groups the tissues of a similar origin and sorts them in ascending order
   reformattedData.sort((x, y) => {
-    const a = TISSUE_COLOR_DICT[x.name];
-    const b = TISSUE_COLOR_DICT[y.name];
+    const a = TISSUE_ORGAN_DICT[x.name];
```

Comment:
do we know that all names in reformattedData will be in TISSUE_ORGAN_DICT? if not, maybe you want to first check that the names are in the DICT and if they're missing, sort them to the end?